### PR TITLE
C# Unit Test Project: Set PublishTrimmed =false even in Release mode.

### PR DESCRIPTION
**Pull request by kmahone**
The unit tests do not work with PublishTrimmed=true. So building the unit test project in Release configuration will result in the tests no working. 

The fix is to set PublishTrimmed=false unconditionally instead of basing it on the Configuration.

### CODE REVIEW

File: dev/VSIX/ProjectTemplates/Desktop/CSharp/UnitTestApp/Properties/PublishProfiles/win-arm64.pubxml [hunk 1]
```diff
@@ -11,9 +11,7 @@ https://go.microsoft.com/fwlink/?LinkID=208121.
     <PublishDir>bin\$(Configuration)\$(TargetFramework)\$(RuntimeIdentifier)\publish\</PublishDir>
     <SelfContained>true</SelfContained>
     <PublishSingleFile>False</PublishSingleFile>
-    <PublishReadyToRun Condition="'$(Configuration)' == 'Debug'">False</PublishReadyToRun>
-    <PublishReadyToRun Condition="'$(Configuration)' != 'Debug'">True</PublishReadyToRun>
-    <PublishTrimmed Condition="'$(Configuration)' == 'Debug'">False</PublishTrimmed>
-    <PublishTrimmed Condition="'$(Configuration)' != 'Debug'">True</PublishTrimmed>
+    <PublishReadyToRun>False</PublishReadyToRun>
```
Comment (Scottj1s) [inline dev/VSIX/ProjectTemplates/Desktop/CSharp/UnitTestApp/Properties/PublishProfiles/win-arm64.pubxml:14]: PublishTrimmed makes sense, but why PublishReadyToRun?  That would make tests run faster.

Comment (kmahone) [reply dev/VSIX/ProjectTemplates/Desktop/CSharp/UnitTestApp/Properties/PublishProfiles/win-arm64.pubxml]: Thanks. I've reverted the change to PublishReadyToRun.

**Review by Scottj1s — APPROVED** [review]

**Comment by Scottj1s** [comment]
thanks!

**Pull request merged**